### ISSUE TITLE
Authentication Advanced Global Settings don't save

### ISSUE THREAD

**Issue by preems1** [Requarks/wiki#1951]
**Describe the bug**
Advanced Global Settings in Admin>Authentication reset to default after being changed and saved.

**To Reproduce**
1. Go to Admin>Authentication
2. Scroll down to Advanced Global Settings
3. Change JWT Audience, Token Expiration, and/or Token Renewal to values of your choice.
4. Click Apply at top of page
5. After note of successful configuration change, navigate to different page
6. Navigate back to Admin>Authentication
7. See that Advanced Global Settings have been reset to default

**Expected behavior**
Advanced Global Settings will be saved

**Host Info (please complete the following information):**
 - OS: Docker
 - Wiki.js version: 2.3.81
- Database engine: postgres:11-alpine

**Additional context**
I have the Keycloak module active but I have experienced this error without Keycloak module.

**Comment by ftm** [comment]
I am also experiencing this, however after doing some digging it looks like it's (at least partially) a UI bug.

When I update the audience from `urn:wiki.js` to `urn:mywiki.example.com` and click Apply, my token is invalidated as expected and I am forced to log back in. After logging back in, the audience field reverts back to `urn:wiki.js`.

However, when I decode the JWT I can see that the audience is correct, and in the database I can see that the audience has indeed been updated:

```
mywiki=# select * from settings where key = 'auth';
 key  |                                      value                                       |        updatedAt         
------+----------------------------------------------------------------------------------+--------------------------
 auth | {"audience":"urn:mywiki.example.com","tokenExpiration":"30m","tokenRenewal":"14d"} | 2020-06-25T21:44:34.807Z
(1 row)
```

If I then click Apply again, the default value of `urn:wiki.js` is saved over my custom audience and the token is once again made invalid.

```
mywiki=# select * from settings where key = 'auth';
 key  |                                      value                                       |        updatedAt         
------+----------------------------------------------------------------------------------+--------------------------
 auth | {"audience":"urn:wiki.js","tokenExpiration":"30m","tokenRenewal":"14d"} | 2020-06-25T21:44:34.807Z
(1 row)
```

**Comment by NGPixel** [comment]
Fixed for 2.5.
